nilearn/_utils/niimg.py

```diff
@@ -0,0 +1,157 @@
+"""
+Neuroimaging file input and output.
+"""
+# Author: Gael Varoquaux, Alexandre Abraham, Philippe Gervais
+# License: simplified BSD
+
+import copy
+import gc
+import collections
+from six import string_types
+
+import numpy as np
+import nibabel
+
+from .numpy_conversions import as_ndarray
+
+
+def _safe_get_data(img):
+    """ Get the data in the image without having a side effect on the
```

Comment:
Sorry, typo (doing too much email), I meant 1.1. Of course we don't update to 2.0. It's way too recent.